.github/workflows/tagPublish.yml

```diff
@@ -0,0 +1,86 @@
+name: package and publish to pypi
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  github-release:
+    name: Create Github release page
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.release.tag_name }}
+      - name: Set release version
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Create Github release page
+        # Create the release page first or the nodejs artifacts upload will fail
```

Comment:
Correct the comment as we are not using nodejs here.

Reply:
Done.